test/credit-card.js

```diff
@@ -102,4 +106,44 @@ describe('Credit Card validations', () => {
       )
     );
   });
+
+  describe('#isHipercard', () => {
+    context('Hipercard', () => {
+      it('returns true', () => {
+        expect(isHipercard(g.generateHipercard().vcc)).to.be.true;
+      });
+    });
+
+    context('Elo', () => {
+      it('returns false', () => {
+        expect(isHipercard(g.generateEloCard().vcc)).to.be.false;
+      });
+    });
+
+    context('Diners Club International', () => {
+      it('returns false', () => {
+        expect(isHipercard(chance.cc({ type: 'Diners Club International' }))).to.be.false;
+      });
+    });
+  });
+
+  describe('#isElo', () => {
+    context('Elo', () => {
+      it('returns true', () => {
+        expect(isElo(g.generateEloCard().vcc)).to.be.true;
+      });
+    });
+
+    context('Elo', () => {
+      it('returns false', () => {
+        expect(isElo(g.generateHipercard().vcc)).to.be.false;
+      });
+    });
+
+    context('Diners Club International', () => {
```

Comment:
The context is `'Diners Club International'` and then you use `chance.cc({ type: 'Visa' })`